uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/Service/LeaveRequestCalendarFeedData.php

```diff
@@ -109,13 +117,24 @@ private function getLeaveTypeLabel($typeId) {
 
     return !empty($absenceTypesList[$typeId]) ? $absenceTypesList[$typeId] : 'Leave';
   }
+
+  /**
+   * Returns the feed config timezone.
+   *
+   * @return string
+   */
+  public function getTimeZone() {
+    return $this->feedConfig->timezone;
+  }
+
   /**
-   * Returns the Feed Config object.
+   * Returns the datetime for this leave feed data class
+   * was instantiated.
    *
-   * @return LeaveRequestCalendarFeedConfig
+   * @return \DateTime
    */
-  public function getFeedConfig() {
-    return $this->feedConfig;
+  public function getInstantiatedDateTime() {
+    return $this->instantiatedDateTime;
```

Comment:
Why was this change necessary? If it's only to help with the test, then we need to find another way to test it. We cannot add new properties to classes only because we need it for a test